src/components/test/FollowersFollowingUsers.test.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import FollowersFollowingUsers from "../FollowersFollowingUsers";
+import { shallow, mount } from "enzyme";
+import { preFabProfiles } from "../../test/testProfiles";
+
+const mockTempUserList = [
+  {
+    profile: preFabProfiles[0],
+    icon: preFabProfiles[0].icon,
+    name: preFabProfiles[0].name,
+    following: true,
+    followsMe: true,
+  },
+  {
+    profile: preFabProfiles[1],
+    icon: preFabProfiles[1].icon,
+    name: preFabProfiles[1].name,
+    following: true,
+    followsMe: false,
+  },
+  {
+    profile: preFabProfiles[2],
+    icon: preFabProfiles[2].icon,
+    name: preFabProfiles[2].name,
+    following: false,
+    followsMe: true,
+  },
+];
+
+describe("Feed", () => {
+  it("renders without crashing", () => {
+    expect(() => {
+      shallow(
+        <FollowersFollowingUsers
+          selectedListTitle="showFollowers"
+          tempUserList={mockTempUserList}
+        />
+      );
+    }).not.toThrow();
+  });
+  it("to match snapshot", () => {
+    const component = mount(
+      <FollowersFollowingUsers
+        selectedListTitle="showFollowers"
+        tempUserList={mockTempUserList}
+      />
+    );
+    expect(component).toMatchSnapshot();
```

Comment:
I'd like to try avoiding using `MatchSnapshot()` where possible. The issue with it is that every time we change the component we have to re-examine the whole snapshot to make sure the test is still working as intended. And if you ever miss something looking back through the snapshot the test stops working as intended.